autoscheduler/scraper/tests/models_tests.py

```diff
@@ -29,3 +30,14 @@ def test_generate_course_id_does_form(self):
         result = generate_course_id("CSCE", "121", "201931")
 
         assert result == expected
+
+class MeetingTests(unittest.TestCase):
+    """ Meeting model tests """
+    def test_can_generate_meeting_id(self):

```

Comment:
```suggestion
    def test_generate_meeting_id_does_form(self):
```

Should start with the system(function) under test , which is `generate_meeting_id` in this case.